powerhome/playbook
# [PLAY-119] Background kit custom colors prop

#### Screens

<img width="1492" alt="image" src="https://user-images.githubusercontent.com/84349455/154736365-66f4a855-d093-488c-a254-90935c5a8838.png">


#### Breaking Changes

No, this PR adds a new prop to the Background kit that allows users to enter a hex value and apply that custom color to their background kit.

#### Runway Ticket URL

[https://nitro.powerhrg.com/runway/backlog_items/PLAY-119](url)

#### How to test this
1. Use the background kit in either a .jsx or .erb file using the `custom_color` prop
2. Pass a 3, 6, or 8 character hex value to the prop
3. Load the page using the kit and check to make sure the background matches the color picker that the hex value was pulled from.

#### Checklist:
